lib/game.rb

```diff
@@ -0,0 +1,113 @@
+require 'pry'
+
+class Game
+
+  attr_accessor :board, :player_1, :player_2, :position
+
+=begin
+    0 | 1 | 2
+  -------------
+    3 | 4 | 5
+  -------------
+    6 | 7 | 8
+=end
+
+
+  WIN_COMBINATIONS = [
+    [0, 1, 2],
+    [3, 4, 5],
+    [6, 7, 8],
+    [0, 3, 6],
+    [2, 5, 8],
+    [1, 4, 7],
+    [0, 4, 8],
+    [6, 4, 2]
+  ]
+
+  def initialize(player_1 = Players::Human.new("X"), player_2 = Players::Human.new("O"), board = Board.new)
+    @player_1 = player_1
+    @player_2 = player_2
+    @board = board
+  end # initialize
+
+
+  ## determines whose turn it is
+  def current_player
+    if board.turn_count.even?
```

Comment:
This looks good, but would also be the perfect opportunity to use a ternary expression! `@board.turn_count.even? ? @player_1 : @player_2`